crates/ark/src/fixtures/dummy_frontend.rs

```diff
@@ -24,6 +23,14 @@ pub struct DummyArkFrontend {
     guard: MutexGuard<'static, DummyFrontend>,
 }
 
+struct DummyArkFrontendOptions {
```

Comment:
It would be helpful to implement `Default` here for self-documentation purpose (and use it in the default paths for clarity / resilience to typos)